app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
+
+def self.from_devise(params)
+    where(email: params.email).first_or_create do |user|
+    user.password = "password"
+    user.password_confirmation = "password"
+    binding.pry
+    User.create!(user)
+  end
```

Comment:
remove this action as it's unnecessary at this stage. We'll work on another feature with the functionality of creating a random password to be sent to the new user